cmd/ci-secret-bootstrap/main_test.go

```diff
@@ -1089,14 +1089,27 @@ func TestConstructSecrets(t *testing.T) {
 					"a-id-3-2": "attachment-name-3-2-value",
 				},
 			),
-			expectedError: fmt.Errorf("failed to find attachment attachment-name-1 in item item-name-2"),
+			expectedError: fmt.Errorf("[failed to find attachment attachment-name-1 in item item-name-2, failed to find field Pull Credentials in item quay.io, failed to find password in item item-name-3]"),
 		},
 	}
 
 	for _, tc := range testCases {
 		t.Run(tc.name, func(t *testing.T) {
-			actual, acutalError := constructSecrets(tc.config, tc.bwClient)
-			equalError(t, tc.expectedError, acutalError)
+			actual, actualError := constructSecrets(tc.config, tc.bwClient)
+			equalError(t, tc.expectedError, actualError)
+			if actualError != nil {
```

Comment:
This PR introduces a change in behaviour in that we continue to try to get everything even when there was an error. I like it because it allows for quicker debugging when a config change introduces more than one issue